README.md

```diff
@@ -31,79 +30,121 @@ A current version can be viewed in Git Markdown from this repository:
 * [Control implementation statements](docs/controls.md)
 * [Appendices](appendices) (incident response, configuration management, regulations, ...)
 
-### Disclaimer
+## Disclaimer
 
-The contents of these pages are provided as an information guide only. They are intended to enhance compliance understanding and are not intended to be used directly as a System Security Plan without agency-specific review.
+The contents of these pages are provided as an information guide only. They are
+intended to enhance compliance understanding and are not intended to be used
+directly as a System Security Plan without agency-specific review
 
 ## Prerequisites
 
-You will need `docker` and `docker-compose` running locally in a `bash` compatible shell. These can be obtained by installing the [Docker Desktop](https://www.docker.com/products/docker-desktop).
+You will need [Python Poetry](https://python-poetry.org/docs/) to run the
+SSP Toolkit in a Python virtual environment. Once you have Poetry
+installed you will be able to run all the commands using the
+format `poetry run [COMMAND]`.
 
-```bash
-docker --version
-docker-compose --version
-```
 
-### Activate your environment
+## Generating the documentation
+
+To update the local Markdown and or to create new exportable files, perform the following steps:
+
+### createfiles
+
+Create/update the frontmatter, components and appendices using [templates](templates) and [keys](keys)
 
-We use the [bowline](https://github.com/CivicActions/bowline/) docker sandbox helper to instantiate local containers with [compliancetools](https://github.com/CivicActions/compliancetools) and [secrender](https://github.com/CivicActions/secrender). The following two commands will ***activate*** your local environment, setting up local aliases for the bowline-exposed commands which are described below.
+#### Example
 
 ```bash
-docker-compose pull
-source activate
+poetry run createfiles -i configuration.yaml -t templates
```

Comment:
No more `-i` option:
```
$ poetry run createfiles -i configuration.yaml -t templates
Usage: createfiles [OPTIONS]
Try 'createfiles --help' for help.

Error: No such option: -i
```

Reply:
Since we were constantly passing the same configuration file, I removed it from the commands. I'll confirm, but it should throw an error if it doesn't exist.